tox.ini

```diff
@@ -26,7 +26,7 @@ commands=pidiff pubtools-ami .
 [testenv:cov]
 usedevelop=true
 commands=
-	pytest --cov-report=html --cov=pubtools {posargs}
+	pytest --cov-report=xml --cov=pubtools {posargs}
```

Comment:
Purpose of the --cov-report=html here was to support reading the report locally, and I'd still want to do that.  I think it'd be reasonable to generate both xml and html outputs then.